### DISCUSSION TITLE
为什么CondColumn中的子Cond间的关系可以是AND

### DISCUSSION THREAD

**Discussion by JasonLi-cn** [apache/doris#8011] [A - General / Q&A]
be/src/olap/olap_cond.h

```
// 所有归属于同一列上的条件二元组，聚合在一个CondColumn上
class CondColumn {
public:
  // ...
private:

    // Conds in _conds are in 'AND' relationship
    std::vector<Cond*> _conds;
};
```

为什么_conds间的关系可以是AND，如果一个SQL是：
`select * from test_table where column1 < 2 or column1 > 6 limit 10`
 则怎么将两个Cond转换成AND关系？

**Comment by morningman** [comment]
`column1 < 2 or column1 > 6` will be in one `Cond`.
